src/privateai_client/components/request_objects.py

```diff
@@ -512,6 +512,41 @@ def fromdict(cls, values: dict):
             )
 
 
+class OCROptions(BaseRequestObject):
+    default_ocr_system = "azure_computer_vision"
+    VALID_OCR_SYSTEM = ["azure_computer_vision", "azure_doc_intelligence", "hybrid", "paddleocr"]
+
+    def __init__(
+        self,
+        ocr_system: str = default_ocr_system,
+    ):
+        self._ocr_system = ocr_system
+
+    @property
+    def ocr_system(self):
+        return self._ocr_system
+
+    @ocr_system.setter
+    def ocr_system(self, var):
+        if self._ocr_system_validator(var):
+            self._ocr_system = var
+
+    def _ocr_system_validator(self, var):
+        if var not in self.VALID_OCR_SYSTEM:
+            raise ValueError(
+                f"OCROptions.ocr_system must be one of {self.VALID_OCR_SYSTEM}, but got {var}"
```

Comment:
Can you format this to not include the brackets?  ', '.join(self.VALID_OCR_SYSTEM)

Reply:
@a-guiducci I addressed the comment and the test passed.